CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ Table of Contents
 
 ### Improvements
 
+* Add `autofocus` as attribute (needs special handling).
```

Comment:
That would be a breaking change, wouldn't it?

Reply:
Iff an HTML contains autofocus, that change needs to be accepted. It surely isn't API breaking.

Reply:
> Breaking Changes when a change requires manual activity at update.

This is the case. While it is not an API change, it requires user activity. See the similarity to tabindex in the unrealeased section.